scheduler/core/builder/schedulerbuilder.py

```diff
@@ -32,7 +37,8 @@ def build_collector(self,
                         end: Time,
                         sites: FrozenSet[Site],
                         semesters: FrozenSet[Semester],
-                        blueprint: CollectorBlueprint) -> Collector:
+                        blueprint: CollectorBlueprint,
+                        program_list: Optional[bytes] = None) -> Collector:
```

Comment:
Good question. It may be used in operations in some cases, but I doubt it. I'll give it a think before I merge.
In any case, the interfaces aren't really compatible if it isn't there, unfortunately.

Reply:
Now that I think about it, there may be cases for whatever reason where an operator in GPP might want to exclude certain programs. Since the interfaces should be the same, we should leave these be, but as you put it here, set to `None` to indicate all programs.